src/garage/torch/_functions.py

```diff
@@ -273,3 +291,36 @@ def product_of_gaussians(mus, sigmas_squared):
     sigma_squared = 1. / torch.sum(torch.reciprocal(sigmas_squared), dim=0)
     mu = sigma_squared * torch.sum(mus / sigmas_squared, dim=0)
     return mu, sigma_squared
+
+
+class TransposeImage(gym.ObservationWrapper):
+    """Transpose observation space for image observation in PyTorch."""
```

Comment:
i dont think a wrapper is the best way to approach this, but i might be wrong.

why not just transpose all of the observations as you receive them? what is the cost of doing this in the differentiable path, vs during sampling (what you do here)?

Reply:
The reason for `TransposeImage` is to transform the gym `observation_space`, and not just the observation itself so that `observation_space.shape` and `observation` is consistent. And it is useful for manipulating dimension and observation input from `get_action` / `get_actions` in Pytorch format. 

And you're right, transforming the observation space is expensive if we do it rapidly. But in fact, we just need to perform once at the beginning when I get the `env`.

Reply:
If I don't transform the observation space, I need to (remember to) transpose the `observation` / `shape` every time in pytorch.